File: samples/android/app/src/main/kotlin/jp/kamusoft/ksdialogs/samples/android/InlineToastViewModel.kt
package jp.kamusoft.ksdialogs.samples.android

import jp.kamusoft.ksdialogs.ToastViewModel

/**
 * インライン経路のカスタム Toast の ViewModel。
 *
 * 中身はこの場で渡すため、このクラスはレジストリに登録しない (core/ADR-0013)。
 *
 * @property message Toast に表示するメッセージ
 */
internal class InlineToastViewModel(val message: String) : ToastViewModel

File: samples/ios/KsDialogsSample/InlineToastViewModel.swift
import KsDialogs

/// インライン経路のカスタム Toast の ViewModel。
///
/// 中身はこの場で渡すため、この型はレジストリに登録しない (core/ADR-0013)。
final class InlineToastViewModel: ToastViewModel {
    /// Toast に表示するメッセージ。
    let message: String

    init(message: String) {
        self.message = message
    }
}
